projects/gloo/api/v1/enterprise/options/extauth/v1/extauth-internal.proto

```diff
@@ -680,6 +680,18 @@ message ExtAuthConfig {
         }
 
     }
+    message PortalAuthConfig {
```

Comment:
actually should we add a comment/document specifically stating this is for **Gloo Gateway** Portal? Since we own 3 portal products now i think (mesh/platform, edge, and gateway)